src/rpc/blockchain.cpp

```diff
@@ -1558,11 +1607,53 @@ RPCHelpMan getblockchaininfo()
     SoftForkDescPushBack(tip, softforks, consensusParams, Consensus::DEPLOYMENT_SEGWIT);
     SoftForkDescPushBack(tip, softforks, consensusParams, Consensus::DEPLOYMENT_TESTDUMMY);
     SoftForkDescPushBack(tip, softforks, consensusParams, Consensus::DEPLOYMENT_TAPROOT);
-    obj.pushKV("softforks", softforks);
+    return softforks;
+}
+} // anon namespace
 
-    obj.pushKV("warnings", GetWarnings(false).original);
-    return obj;
-},
+static RPCHelpMan getdeploymentinfo()
+{
+    return RPCHelpMan{"getdeploymentinfo",
+        "Returns an object containing various state info regarding soft-forks.",
```

Comment:
fd82613 It looks like you are writing "softfork" in the RPC documentation, and "soft fork" in the release note, so maybe standardize on one of these two throughout.

```suggestion
        "Returns an object containing various state info regarding softforks.",
```

Reply:
buried deployments aren't soft forks, so it might be better to call this "deployments"

Reply:
Switched from `soft[ -]*forks` in #24187. Note that rpc/miner.cpp calls them "softforks" in its RPC docs...

Reply:
Yeah, I am not suggesting to change existing fields, but for new ones we can pick a good name from the beginning.